toolkit/tools/graphpkgfetcher/graphpkgfetcher.go

```diff
@@ -68,6 +68,16 @@ func main() {
 		logger.Log.Fatalf("unable to read toolchain manifest file '%s': %s", *toolchainManifest, err)
 	}
 
+	dependencyGraph.InitLookup()
```

Comment:
I'd like to sequence this PR behind my delta downloader PR (https://github.com/microsoft/CBL-Mariner/pull/5595). I don't think there will be any major conflicts but that PR is an important component in an urgent work stream and I'd like to make sure it gets in without issues.